pkg/controller/mysqlcluster/mysqlcluster_controller_test.go

```diff
@@ -95,14 +96,16 @@ var _ = Describe("MysqlCluster controller", func() {
 				},
 			}
 
-			cluster = &api.MysqlCluster{
+			theCluster := &api.MysqlCluster{
```

Comment:
you can directly define `cluster`  here and use `cluster.Unwrap()` when needed.
```suggestion
			cluster := mysqlcluster.New(&api.MysqlCluster{
```